primefaces/src/main/resources/META-INF/resources/primefaces/calendar/3-jquery.ui.datepicker.css

```diff
@@ -0,0 +1,266 @@
+/*! jQuery UI - v1.13.1 - 2022-06-21
+* http://jqueryui.com
+* Includes: core.css, datepicker.css
+* Copyright jQuery Foundation and other contributors; Licensed MIT */
+
+/* Layout helpers
+----------------------------------*/
+.ui-helper-hidden {
```

Comment:
layout helpers are not related to datepicker only

Reply:
OK this came down by default when I downloaded DatePicker by itself from Jquery UI let me remove.